services/collaboration/pkg/command/server.go

```diff
@@ -86,7 +86,6 @@ func Server(cfg *config.Config) *cli.Command {
 						v = make(map[string]map[string]string)
 					}
 					appURLs.Store(v)
-
 					// register the app provider
```

Comment:
get rid of this change
```suggestion
appURLs.Store(v)

// register the app provider
```

Reply:
done